### PR TITLE
WIP: Add smallipop to Reader for glossary definitions

### PR DESCRIPTION
**Pull request by hoverduck**
#### Changes proposed in this Pull Request

* Adds an embed handler to the Reader to enable tooltips for glossary term definitions

#### Testing instructions

* Load a Reader URL and various streams as an a11n and verify the terms are displayed appropriately

### CODE REVIEW

File: client/components/embed-container/index.jsx [hunk 1]
```diff
@@ -34,6 +35,8 @@ const SLIDESHOW_URLS = {
 	SPINNER: `https://s0.wp.com/wp-content/mu-plugins/shortcodes/img/slideshow-loader.gif${ cacheBustQuery }`,
 };
 
+const hovercard_URL_Prefix = 'https://s0.wp.com/wp-content/plugins/reader-smallipop/';
```
Comment (blowery) [inline client/components/embed-container/index.jsx]: all caps for constants with `_` in them pretty please

File: client/components/embed-container/index.jsx [hunk 2]
```diff
@@ -148,6 +151,40 @@ function embedTumblr( domNode ) {
...
+let smallipopLoader;
+function embedSmallipop( domNode ) {
+	debug( 'processing smallipop for', domNode );
+	if ( smallipopLoader ) {
+		return;
+	}
+
+	// TODO - put the CSS/JS files in a better location
+	loadjQueryDependentScript( hovercard_URL_Prefix + 'jquery.smallipop.js', function() {
+		smallipopLoader = true;
```
Comment (blowery) [inline client/components/embed-container/index.jsx]: this set should be outside the load callback to prevent more than one attempt at loading

**Comment by blowery** [comment]
Could we add `cursor: help` to `.smallipop-instance .sipArrow` in `wp-content/plugins/reader-smallipop/jquery.smallipop.custom.css`? That would help with the pointer flashing from the ? over to a plain pointer on hover.

**Comment by nickmomrik** [comment]
> Could we add `cursor: help` to `.smallipop-instance .sipArrow` in `wp-content/plugins/reader-smallipop/jquery.smallipop.custom.css`? That would help with the pointer flashing from the ? over to a plain pointer on hover.

Good idea! Done.

**Comment by nickmomrik** [comment]
Found an issue where hovercards are not working for comments on posts with a lot of them (paged) or at least that's only what seems to be affected.

**Pull request closed without merging**